tools-ui/src/main/java/com/hedera/hashgraph/client/ui/MainController.java

```diff
@@ -126,9 +126,9 @@
 import static org.zeroturnaround.zip.commons.FileUtils.copyDirectory;
 import static org.zeroturnaround.zip.commons.FileUtils.deleteDirectory;
 
-public class Controller implements Initializable, GenericFileReadWriteAware {
+public class MainController implements Initializable, GenericFileReadWriteAware {
```

Comment:
somehow this is disarranged. In main branch `MainController` is already used. Based on that it should not be displayed as a change in the PR

Reply:
Seems that a squash merge or perhaps rebasing can cause this. After changing the target of the merge to another branch and back to main, the issue seems to be gone.